Common/src/main/java/com/tabnineCommon/chat/commandHandlers/chatSettings/ChatSettings.kt

```diff
@@ -0,0 +1,18 @@
+package com.tabnineCommon.chat.commandHandlers.chatSettings
+
+import com.google.gson.Gson
+import com.intellij.ide.util.PropertiesComponent
+
+private const val CHAT_SETTINGS_KEY = "com.tabnine.chat.settings"
+
+data class ChatSettingsProps(val isTelemetryEnabled: Boolean?)
```

Comment:
```suggestion
data class ChatSettingsProps(val isTelemetryEnabled: Boolean? = null)
```

this way you don't have to pass `null` to the constructor (`ChatSettingsProps(null)` -> `ChatSettingsProps()`)